components/UI/Buttons/ContainedButton/index.tsx

```diff
@@ -1,11 +1,18 @@
 
 import React, { memo } from 'react'
 import { makeStyles } from '@material-ui/core/styles'
-import { Button, CircularProgress } from '@material-ui/core'
+import { Button, CircularProgress, PropTypes } from '@material-ui/core'
 import clsx from 'clsx'
 
 import ButtonLink from 'components/UI/Buttons/ButtonLink'
 
+type ContainedButtonProps = {
+  classes?: any;
+  color?: PropTypes.Color;
+  href?: string;
+  loading?: boolean;
```

Comment:
`Boolean` could have a default of `false` instead of being `optional` `null`

Reply:
Then we will need to pass loading props and it is required. That's why loading needs to be optional here.